lib/developer_portal/app/views/developer_portal/docs.html.liquid

```diff
@@ -15,8 +15,9 @@
 
 <script type="text/javascript">
   (function () {
-    var url = "{{spec.url}}";
-    var serviceEndpoint = "{{spec.api_product_production_public_base_url}}"
-    SwaggerUI({ url: url, dom_id: "#swagger-ui-container" }, serviceEndpoint);
+    const url = "{{spec.url}}";
+    const serviceEndpoint = "{{spec.api_product_production_public_base_url}}"
+    const oauth2RedirectUrl = window.location.origin + "/oauth2-redirect.html"
+    SwaggerUI({ url: url, dom_id: "#swagger-ui-container", oauth2RedirectUrl: oauth2RedirectUrl }, serviceEndpoint);
```

Comment:
If I understood it correctly, only existing portals having the API docs under a complex path like `/api/docs/echo` need this change, right? For instance in my local, where the docs are under `/docs`, I removed both `oauth2RedirectUrl` and `serviceEndpoint` and it works anyway

Reply:
Yes, that's right.

In fact, it is not required to include `oauth2RedirectUrl` piece in this template, because by default it's also under `/docs`, but I guess as the path can be changed very easily, it might be very confusing, if it works with `/docs`, but immediately stops working if the user changes the path of the docs page.

I think it's good to use as an example.